src/status_im/transport/shh.cljs

```diff
@@ -151,6 +151,32 @@
                              (on-success resp)
                              (on-error err))))))
 
+(defn add-sym-keys-batch
+  [{:keys [web3 keys on-success on-error]}]
+  (let [batch    (.createBatch web3)
+        results  (atom [])
+        total    (count keys)
+        counter  (atom 0)
+        callback (fn [chat-id sym-key err resp]
+                   (swap! counter inc)
+                   (if err
+                     (on-error err)
+                     (swap! results conj {:chat-id    chat-id
+                                          :sym-key    sym-key
+                                          :sym-key-id resp}))
+                   (when (= @counter total)
+                     (log/debug :success :add-sym-key-batch)
+                     (on-success @results)))]
+    (log/debug :add-sym-key-batch total)
```

Comment:
leftover

Reply:
it is a valid debug statement, i would prefer to keep it